project/src/main/webapp/components/comment-component.js

```diff
@@ -0,0 +1,52 @@
+import {html, LitElement} from 'https://unpkg.com/@polymer/lit-element/lit-element.js?module';
+
+export class CommentComponent extends LitElement {
+  static get properties() {
+    return {
+      placeholder : {type: String},
+      name : {type: String},
+      time : {type: String}
+    };
+  }
+
+  constructor() {
+    super();
+    this.placeholder = 'Write comment...';
+    this.name ='';
+    this.time = '';
+  }
+
+  // Remove shadow DOM so styles are inherited
+  createRenderRoot() {
+    return this;
+  }
+
+  getServletData() {
+    fetch('/UserHome').then((response) => response.json()).then((documentsData) => { 
```

Comment:
This is a place holder to show what comments are suppose to look like, servlet data will not be needed in the final comment-component